axum-macros/tests/from_request/fail/generic_without_via.stderr

```diff
@@ -12,15 +12,24 @@ warning: unused import: `Extension`
   |
   = note: `#[warn(unused_imports)]` on by default
 
-error[E0277]: the trait bound `fn(Extractor<()>) -> impl Future<Output = ()> {foo}: Handler<_, _>` is not satisfied
+error[E0599]: no function or associated item named `new` found for struct `Router<Body>` in the current scope
+  --> tests/from_request/fail/generic_without_via.rs:10:21
+   |
+10 |     Router::<Body>::new().route("/", get(foo));
+   |                     ^^^ function or associated item not found in `Router<Body>`
+   |
+   = note: the function or associated item was found for
+           - `Router<(), B>`
```

Comment:
This seems unintentional, looks like the test should be updated to use `Router::<_, Body>`.

Reply:
oh yeah good catch!